happi/tests/test_cli.py

```diff
@@ -618,6 +616,46 @@ def test_load(
         assert "Creating shell with devices" in caplog.text
 
 
+def test_load_multiple_args(
+    caplog: pytest.LogCaptureFixture,
+    client: happi.client.Client,
+    happi_cfg: str,
+    runner: CliRunner
+):
+    # try to load the item
+    devices = {}
+    devices['tst_base_pim'] = client.load_device(name='tst_base_pim')
+    devices['tst_base_pim2'] = client.load_device(name='tst_base_pim2')
+    with mock.patch.object(IPython, 'start_ipython') as m:
+        _ = runner.invoke(
+            happi_cli, ['--path', happi_cfg, 'load', 'tst_base_pim', 'tst_base_pim2']
+        )
+        m.assert_called_once_with(argv=['--quick'], user_ns=devices)
+    with caplog.at_level(logging.INFO):
+        assert "Creating shell with devices" in caplog.text
+
+
+def test_load_glob_args(
+    caplog: pytest.LogCaptureFixture,
+    client: happi.client.Client,
+    happi_cfg: str,
+    runner: CliRunner
+):
+    # try to load the item
+    results = client.search_regex(name='tst_base_.*')
+
+    devices = {}
+    devices['tst_base_.*'] = [res.item['name'] for res in results]
```

Comment:
So what you're doing here is building a namespace that we expect in the ipython session.  A namespace being a mapping from variable name (key) to python object (value).   

`tst_base_.*` is not a valid python variable name, so this won't match what's in the mock ipython namespace.  This is why we enumerate the devices in previous tests.

Reply:
I think I finally figured out how this works lol. Committed the changes.